docs/src/terminology.md

```diff
@@ -220,6 +231,12 @@ A computer participating in a [cluster](#cluster).
 
 The number of [validators](#validator) participating in a [cluster](#cluster).
 
+## optimistically confirmed block
+
+A [block](#block) which got the [ledger votes](#ledger-vote) from
+[validators](#validator) holding more than 2/3 of total [stake](#stake) in a
+[cluster](#cluster). The votes may or may not have been confirmed to a block.
```

Comment:
Think we can simplify this to: `A block that has received supermajority stake-weighted votes should not get rolled back given < 5% malicious. Details can be found at [Optimistic Confirmation](optimistic_confirmation.md#Primitives)`

Reply:
Done